### PR TITLE
Music: no project for video level

### PR DESCRIPTION
**Pull request by breville**
This uses the functionality added in https://github.com/code-dot-org/code-dot-org/pull/52567 to avoid using a project/channel on the `StandaloneVideo` level that appears alongside `Music` levels.

### CODE REVIEW

File: apps/src/labs/labRedux.ts [hunk 1]
```diff
@@ -97,10 +97,7 @@ export const setUpWithLevel = createAsyncThunk(
       levelProperties.isProjectLevel,
       false
     );
-    const disableProjects = convertOptionalStringToBoolean(
-      levelProperties.disableProjects,
-      false
-    );
+    const disableProjects = !!levelProperties.disableProjects;
```
Comment (sanchitmalhotra126) [inline apps/src/labs/labRedux.ts]: Does this still work correctly if the boolean is stringified on the backend ("false")?

Comment (breville) [reply apps/src/labs/labRedux.ts]: I tested end-to-end and the boolean isn't stringified in this flow.  (I didn't work properly until I made this change.). Would we expect a string to be used instead?

Comment (sanchitmalhotra126) [reply apps/src/labs/labRedux.ts]: Hmm, I'm not sure - I'm noticing that on some levels (for example "New Music Lab Project.level"), boolean values are stored as strings (ex. `"is_project_level": "true"`). Maybe a side effect of properties being serialized from levelbuilder? We might need to update `convertOptionalStringToBoolean` to account booleans as well as strings.

Comment (breville) [reply apps/src/labs/labRedux.ts]: I've updated that function to handle both types now.

File: apps/src/types/utils.ts [hunk 2]
```diff
@@ -1,11 +1,16 @@
 // Helper for converting a string boolean ('true' or 'false') to a boolean.
-// Many of our apis will return strings rather than booleans.
+// However, if an actual boolean is passed in, it's returned directly.
+// If undefined is passed in, then the provided defaultValue is returned.
+// Some of our server APIs will return strings rather than booleans.
 export function convertOptionalStringToBoolean(
-  str: 'true' | 'false' | undefined,
+  value: 'true' | 'false' | true | false | undefined,
```
Comment (sanchitmalhotra126) [inline apps/src/types/utils.ts]: nit: can this be `'true' | 'false' | boolean | undefined`?

**Review by sanchitmalhotra126 — APPROVED** [review]

File: apps/src/types/utils.ts (hunk 2 above)
Comment (breville) [reply apps/src/types/utils.ts]: Sure thing.

**Review by molly-moen — APPROVED** [review]

**Pull request merged**